tools/c7n_gcp/c7n_gcp/resources/cloudtasks.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+import jmespath
+from c7n_gcp.actions import MethodAction
+from c7n_gcp.provider import resources
+from c7n_gcp.query import QueryResourceManager, TypeInfo, ChildResourceManager, ChildTypeInfo
+
+from c7n.utils import local_session, type_schema
+
+
+@resources.register('cloudtasks-location')
+class CloudTasksLocation(QueryResourceManager):
+    """GCP resource: https://cloud.google.com/tasks/docs/reference/rest/v2/projects.locations
+    This one is present only as the parent of other resources.
+    """
+    def get_resource_query(self):
+        return {'name': 'projects/%s' % local_session(self.session_factory).get_default_project()}
```

Comment:
this is the fifth+ time i've seen this and commented, please let's stop doing parent queries for static information, make a data structure that the code can reference, not roundtrip for parents.

this should not be a resource.